src/utils/client-factory.ts

```diff
@@ -54,6 +56,12 @@ const clientFactory = <T extends ClientType>(
             return new ApolloClient({
                 cache: new InMemoryCache(),
                 uri,
+                headers,
+                link: new HttpLink({
+                    uri,
+                    headers,
+                    fetchOptions: { method: 'GET' },
+                }),
```

Comment:
Override the default POST method and use the GET method instead
https://www.apollographql.com/docs/react/networking/advanced-http-networking